### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by cston**
@jaredpar, @pilchie, @ManishJayaswal

### CODE REVIEW

File: src/Compilers/CSharp/Portable/Errors/ErrorCode.cs [hunk 1]
```diff
@@ -771,7 +771,7 @@ internal enum ErrorCode
         ERR_ParamsCantBeRefOut = 1611,
         ERR_ReturnNotLValue = 1612,
         ERR_MissingCoClass = 1613,
-        ERR_AmbigousAttribute = 1614,
+        ERR_AmbiguousAttribute = 1614,
```
Comment (jaredpar) [inline src/Compilers/CSharp/Portable/Errors/ErrorCode.cs]: Hmm, that's not good to have in an API.  Luckily it's a private one.  Have we thought about a spell checker for public APIs?

**Comment by Pilchie** [comment]
LGTM

**Comment by MattGertz** [comment]
Rejected 6/9 by ML Shiproom.

**Comment by cston** [comment]
@dotnet-bot retest this please

**Pull request merged**